prow/spyglass/lenses/junit/template.html

```diff
@@ -156,7 +156,7 @@
       {{range .Skipped}}
         {{$firstTest := index .Junit 0}}
         <tr>
-          <td class="mdl-data-table__cell--non-numeric test-name">{{$firstTest.ClassName}}: {{$firstTest.Name}}</td>
+          <td class="mdl-data-table__cell--non-numeric test-name">{{$firstTest.ClassName}}: {{$firstTest.Name}}<br/>{{$firstTest.Skipped}}</td>
```

Comment:
I wonder if it would be useful to include an explicit `Reason: ` prefix? Up to you.

```suggestion
          <td class="mdl-data-table__cell--non-numeric test-name">{{$firstTest.ClassName}}: {{$firstTest.Name}}<br/>Reason: {{$firstTest.Skipped}}</td>
```

/hold

Reply:
@petr-muller, thank you for your feedback. I've revised the PR to include the suggestion for an explicit `Reason:` prefix.

Adding context to the new changes:

- There are cases (mainly) where skipped tests do not pass any skipped reason string to the `e2eskipper.Skipf` method. So I added an if/else conditional check in the go template to handle the scenarios of an empty skipped reason (which required the addition of a helper function `SkippedReason()` in the`lens.go` file)
- I created the new function because:
    - directly using  `{{ if eq $firstTest.Skipped "" }}` in the go template didn't work, throwing incompatible string & pointer type comparisons. [Here's](https://stackoverflow.com/questions/35609903/dereferencing-pointers-in-a-golang-text-template) a similar example.
    - Or in case of using`{{if $firstTest.Skipped}}`, it is never `False` (since it is always pointing to an empty skipped reason, therefore the conditional didn't move). 

The updated result is as follows: 

![Screenshot 2022-02-14 at 11 07 36 AM](https://user-images.githubusercontent.com/30499743/153806696-78076d33-94fc-472c-84c2-b52ed07d7cb3.png)